### ISSUE TITLE
Nice error page in debug mode

### ISSUE THREAD

**Issue by szabgab** [ufobat/Bailador#106]
It would be nice if Bailador would catch any exceptions during its processing and instead of "Internal Server Error" it would give a nice error page. (e.g. like Flask does)

**Comment by ufobat** [comment]
what do you think of the idea that the error page could be a template as well. all we need to do is ship a "hardcoded"  500.debug template.

there would be 4 different cases

* debug mode  == 1 and no $!location/views/500.debug available
* debug mode == 1 and a 500.debug is avilable  (so somone could still create his own default error page
* debug mode == 0 and no 500.xx => still the Inernal server error
* debug mode == 0 and a 500.xx evailable => template should be used

**Comment by szabgab** [comment]
I am working on a template for the 500 page. I am not sure what would we want to show in debug mode if there is a user-created 500 template. Actually I am not even sure we want to stay with a simple template for 500 errors. Flask for example allows you to create a "virtual route" which is called in case there is a 500 error. That way the developer can do more than just show a plain page.

**Comment by JJ** [comment]
Wouldn't it be better if this was an external file, instead of hard-coding it? Better user configuration, too.

**Comment by ufobat** [comment]
I agree with you, what is hardcoded could/should probably be in the resources. This is the last resort, if there are error templates in your application, I think they're named 500.xx, you would use them. So from the configuration/usability perspective I'd say it is okay. no?
